### DISCUSSION TITLE
question - tulip.set_time() does not work on tulip desktop

### DISCUSSION THREAD

**Discussion by diflouroethane** [shorepine/tulipcc#307] [General]
when I run tulip.set_time() in the REPL, it says no module named 'ntptime'

thank you for the help!

**Comment by bwhitman** [comment]
You don't need `set_time` on Desktop, as it "knows" the time of the host system. We probably should not make those functions (like `wifi` as well) available on Desktop.

**Comment by diflouroethane** [comment]
Ah. That makes sense. Thanks!
